src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -222,4 +222,57 @@ describe('Reimbursement Requests', () => {
       expect(prisma.reimbursement_Request.update).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Delivered Tests', () => {
+    test('Mark as delivered fails for non submitter', async () => {
+      jest.spyOn(prisma.reimbursement_Request, 'findUnique').mockResolvedValue({ ...GiveMeMyMoney, dateDelivered: null });
```

Comment:
dont need spread operator here as give me my money by default has date delivered to be null i believe